### PR TITLE
Add host with ipv6

### PR DESCRIPTION
**Pull request by vkondula**
You can add host using ipv6
Always try to resolve FQDN. Before if you created host
 with domain name but not fully qualified it was stored
 as fqdn
Add ability to find default ipv6 for gateway.
Add ability to find ipv6 by interface
fixes: #56

### CODE REVIEW

**Comment by vkondula** [comment]
reverts: #62, reason in commit msg

**Comment by lukas-bednar** [comment]
Looks good to me :+1:

@myakove can you take a look at it?

File: rrmngmnt/network.py [hunk 1]
```diff
@@ -276,6 +294,29 @@ def find_ip_by_int(self, interface):
...
+    def find_ipv6_by_int(self, interface):
+        """
+        Find host global ipv6 by interface or Bridge name
+
+        :param interface: interface to get ipv6 from
+        :type interface: string
+        :return: IP or None
+        :rtype: string or None
+        """
+        out = self._cmd(["ip", "addr", "show", interface])
```
Comment (myakove) [inline rrmngmnt/network.py]: You can use -6 here too
ip -6 addr show

File: rrmngmnt/network.py [hunk 2]
```diff
@@ -205,6 +205,24 @@ def find_default_gw(self):
...
+        :return: default gateway
+        :rtype: string
+        """
+        out = self._cmd(["ip", "-6", "route"]).splitlines()
+        for i in out:
+            if re.search("default", i):
+                default_gw = re.findall(
+                    r'(?<=\s)[0-9a-fA-F:]{3,}(?=\s)', i
+                )
+                if netaddr.valid_ipv6(default_gw[0]):
```
Comment (myakove) [inline rrmngmnt/network.py]: Do you need this check since we get the IP from  ip -6 command?

Comment (vkondula) [inline rrmngmnt/network.py]: I would prefer to keep it there, if the regex fails to match the address correctly.

**Pull request merged**